### PR TITLE
Transaction reorg

### PR DESCRIPTION
**Pull request by maria-robobug**
Updating some of the shared partials for transactions.

These changes are used in https://github.com/couchbase/docs-sdk-java/pull/340

### CODE REVIEW

File: modules/shared/partials/acid-transactions.adoc [hunk 1]
```diff
@@ -1,84 +1,101 @@
 = ACID Transactions
 
 
+//-------------------------- HOWTO PARTIALS ------------------------
 
 // tag::intro[]
-This document presents a practical HOWTO on using Couchbase transactions, following on from our xref:7.1@server:learn:data/transactions.adoc[transactions documentation].
+Couchbase Distributed xref:7.1@server:learn:data/transactions.adoc#overview[ACID (atomic, consistent, isolated, and durable)] Transactions allow applications to perform a series of database operations as a single unit -- meaning operations are either committed together or all undone.
+Transactions are distributed and work across multiple documents, scopes, and collections, which can reside on multiple nodes.
```
Comment (programmatix) [inline modules/shared/partials/acid-transactions.adoc]: Buckets too.

File: modules/shared/partials/acid-transactions.adoc [hunk 2]
```diff
@@ -1,84 +1,101 @@
...
 // end::intro[]
 
 
-
-
 // tag::requirements[]
-* NTP should be configured so nodes of the Couchbase cluster are in sync with time.
-* The application, if it is using xref:concept-docs:xattr.adoc[extended attributes (XATTRs)], must avoid using the XATTR field `txn`, which is reserved for Couchbase use.
```
Comment (programmatix) [inline modules/shared/partials/acid-transactions.adoc:14]: Is there a reason to ditch these two?  The second is a real edge case, but NTP is v important.

File: modules/shared/partials/acid-transactions.adoc [hunk 3]
```diff
@@ -1,84 +1,101 @@
...
-== Creating a Transaction
-
-A core idea of Couchbase transactions is that an application supplies the logic for the transaction inside a lambda, including any conditional logic required, and the transaction is then automatically committed.
-If a transient error occurs, such as a temporary conflict with another transaction, then the transaction will rollback what has been done so far and run the lambda again.
-The application does not have to do these retries and error handling itself.
-
-Each run of the lambda is called an `attempt`, inside an overall `transaction`.
+To create a transaction, an application must supply its logic inside a `{lambda}`, including any conditional logic required.
+Once the {lambda} has successfully run to conclusion, the transaction will be automatically committed.
+If at any point an error occurs, the transaction will rollback and the {lambda} will be run again.
```
Comment (programmatix) [inline modules/shared/partials/acid-transactions.adoc]: The lambda _may_ run again.  Some errors we fast fail on, some we retry.

File: modules/shared/partials/acid-transactions.adoc [hunk 4]
```diff
@@ -419,6 +394,7 @@ Currently, Distributed ACID Transactions are available for:
 * The xref:2.4@go-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Go SDK].
 * The xref:3.3@java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[Java SDK].
 * The xref:4.0@nodejs-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[node.js SDK].
+<<<<<<< Updated upstream
```
Comment (programmatix) [inline modules/shared/partials/acid-transactions.adoc]: Intended?

**Review by programmatix — CHANGES_REQUESTED** [review]
Is there anywhere I can view a staged version of the new transactions docs?

**Comment by maria-robobug** [comment]
> Is there anywhere I can view a staged version of the new transactions docs?

@programmatix yep, in this PR: https://github.com/couchbase/docs-sdk-java/pull/340

File: modules/shared/partials/acid-transactions.adoc (hunk 4 above)
Comment (maria-robobug) [reply modules/shared/partials/acid-transactions.adoc]: Hah, nope, let me double check why I got that in there and clean this up.

File: modules/shared/partials/acid-transactions.adoc (hunk 2 above)
Comment (maria-robobug) [reply modules/shared/partials/acid-transactions.adoc:14]: I just pulled these out of common, and decided to put them in the individual SDKs, so this is still documented.

File: modules/shared/partials/acid-transactions.adoc [hunk 5]
```diff
@@ -276,138 +366,18 @@ The default settings are tuned to find expired transactions reasonably quickly,
...
 An application may cleanup transactions created by another application.
 
-It is important to understand that if an application is not running, then cleanup is not running.
+NOTE: It is important to understand that if an application is not running, then cleanup is not running.
 This is particularly relevant to developers running unit tests or similar.
 // end::cleanup[]
 
 
 
-// tag::query-intro[]
```
Comment (RichardSmedley) [inline modules/shared/partials/acid-transactions.adoc:291]: Isn't this consumed by the .NET version, too?
Will the merge be waiting for the other SDKs to get the Java doc ported?

Comment (maria-robobug) [reply modules/shared/partials/acid-transactions.adoc:291]: Yep, just wanted to have the ok for Java first so I don't have to go back and forth.
Porting all SDKs to this is the next step.

Once all of SDKs are done I will put these on staging, check they're ok and then go to prod.

**Pull request merged**